### ISSUE TITLE
Cannot open Baggins context menu on retail WoW

### ISSUE THREAD

**Issue by Elberet** [doadin/Baggins#128]
On a fresh installation of retail WoW and with no addons besides Baggins, BugSack and BugGrabber, right-clicking anywhere in a Baggins bag results in this LUA error:

```
Baggins/Baggins-Options.lua:3756: attempt to call global 'EasyMenu' (a nil value)
[string "@Baggins/Baggins-Options.lua"]:3756: in function `DoBagMenu'
[string "@Baggins/Baggins.lua"]:2697: in function <Baggins/Baggins.lua:2693>
```

**Comment by doadin** [comment]
yea I noticed this and the changes mentioned here: https://warcraft.wiki.gg/wiki/Patch_11.0.0/API_changes, I havnt had a ton of time to look into but also when I did didn't seem like a easy change? Not sure.

**Comment by doadin** [comment]
EasyMenu at least for now still exists in current classic versions so this is a retail only needed change.

**Comment by Elberet** [comment]
From a quick read, as of Patch 11.0.0, `UIDropDownMenu` has been deprecated, and `EasyMenu` - which was built on top of the former - is just gone. The wiki links to [this guide](https://www.townlong-yak.com/framexml/latest/Blizzard_Menu/11_0_0_MenuImplementationGuide.lua), which explains how the new menu system works.

**Comment by doadin** [comment]
should be fixed: 01d3f1653e34bcb27e93994909d032ff1927a8dd , I had a working bit locally so hopefully I did it all correctly for release.
